pages/docs/reference/whatsnew1420.md

```diff
@@ -0,0 +1,348 @@
+
+This change of structure breaks the compatibility between projects with [hierarchical project structure](mpp-share-on-platforms.html#share-code-on-similar-platforms).
+If a multiplatform project and a library it depends on both have the hierarchical project structure, then you need to update
+them to Kotlin 1.4.20 or higher simultaneously. Libraries published with Kotlin 1.4.20 are not available for using from
+project published with earlier versions.
+
+Projects and libraries without the hierarchical project structure remain compatible.
+
+[**Back to top**](#)
+
+## CocoaPods support
+
+### Improved task execution
+
+In this release, we significantly improved task execution flow. For example, if you add a new CocoaPods dependency,
+existing dependencies are not rebuilt. Adding an extra target also doesn't affect rebuilding dependencies for existing ones.
+
+### Extended DSL
+
+In 1.4.20, we extend the DSL of adding [CocoaPods](native/cocoapods.html) dependencies to your Kotlin project.
+
+In addition to local Pods and Pods from the CocoaPods repository, you can add dependencies on the following types of libraries:
+* A library from a custom spec repository.
+* A remote library from a Git repository.
+* A library from an archive (also available by arbitrary HTTP address).
+* A static library.
+* A library with custom cinterop options.
+
+Learn more about adding CocoaPods dependencies in Kotlin projects [here](native/cocoapods.html#add-dependencies-on-pod-libraries).
+Find examples in the [Kotlin with CocoaPods sample](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Updated integration with Xcode
+
+To work correctly with Xcode, Kotlin requires some Podfile changes:
+
+* If your Kotlin Pod has any Git, HTTP, or specRepo pod dependency, you should also specify it in the Podfile.
+* When you add a library from the custom spec, you also should specify the [location](https://guides.cocoapods.org/syntax/podfile.html#source)
+    of specs at the beginning of your Podfile.
+
+Now integration errors have a detailed description in IDEA. So if you have problems with your Podfile, you will immediately know how to fix them.
+
+Learn more about creating Kotlin pods [here](native/cocoapods.html#use-a-kotlin-gradle-project-as-a-cocoapods-dependency).
+
+[**Back to top**](#)
+
+## Standard library
+
+### Extensions for java.nio.file.Path
+
+> Extensions for `java.nio.file.Path` are [experimental](evolution/components-stability.html) in 1.4.20. They may change incompatibly in future versions.
+{:.note}
+
+Now the standard library provides experimental extensions for `java.nio.file.Path`.
+Working with the modern JVM file API in an idiomatic Kotlin way is now similar to working with `java.io.File` extensions
+from the `kotlin.io` package.
+
+The extensions are available in the `kotlin.io.path` package in the `kotlin-stdlib-jdk7` module.
+In order to use them, you need to opt-in to the experimental annotation `@ExperimentalPathApi`.
```

Comment:
@p7nov doesn't seem to be your sentence. You like removing unnecessary words such as `in order to` and `you need to` :)

Reply:
Caught me =)